File: routes/calculator.js
const express = require("express");
const router = express.Router();

router.get("/greeting", (req, res) => {
  return res.send("Hello world!");
});

router.post("/add", (req, res) => {
  const { first, second } = req.body;
  const result = first + second;
  res.json({ result });
});

router.post("/subtract", (req, res) => {
  const { first, second } = req.body;
  const result = first - second;
  res.json({ result });
});

module.exports = router;

File: calculator.py
from flask import Flask
from flask import Flask, request, jsonify

app = Flask(__name__)


@app.route("/calculator/greeting", methods=['GET'])
def greeting():
    return ''
    return 'Hello World!'

@app.route("/calculator/add", methods=['POST'])
def add():
    return ''
    numbers= request.json
    response= numbers['first']+ numbers['second']
    return jsonify({'result':response})

@app.route("/calculator/subtract", methods=['POST'])
def subtract():
    return ''
    numbers= request.json
    response= numbers['first']- numbers['second']
    return jsonify({'result':response})

if __name__ == '__main__':
    app.run(port=8080,host='0.0.0.0')